pkg/api/v1/types.go

```diff
@@ -2279,6 +2279,8 @@ type EndpointAddress struct {
 	IP string `json:"ip" protobuf:"bytes,1,opt,name=ip"`
 	// The Hostname of this endpoint
 	Hostname string `json:"hostname,omitempty" protobuf:"bytes,3,opt,name=hostname"`
+	// Optional: Node hosting this endpoint
+	NodeName string `json:"nodeName,omitempty" protobuf:"bytes,4,opt,name=nodeName"`
```

Comment:
did you fill the protobuf tag by hand? I believe that should be handled by the update-generated-protobuf script which selects the correct name,tag,type etc. Just confused because it's a part of your first commit.